exportlib.php

```diff
@@ -84,47 +84,41 @@ public function create_file($courseid, $feedbackid) {
 }
 
 /**
- * @author Felix Di Lenarda
- * rankings CSV export class
+ * Export feedback data for an entire feedbacj.
```

Comment:
... entire feedbac**k**.

Reply:
Geändert in: 133d532afd97506c96e5730ab0acca9161b6b022